### PR TITLE
Bugfix and colors

### PR DESCRIPTION
**Pull request by blazf**

### CODE REVIEW

**Review by rupnikj — APPROVED** [review]

File: src/glib/base/ut.cpp [hunk 1]
```diff
@@ -129,12 +129,45 @@ void TStdErrNotify::OnStatus(const TStr& MsgStr){
...
-		NowTm.GetHMSTColonDotStr(true, false).CStr(), 
+	Notify->OnStatus(TStr::Fmt("[%s %s] %s",
+		NowTm.GetYMDDashStr().CStr(),
+		NowTm.GetHMSTColonDotStr(true, false).CStr(),
 		MsgStr.CStr()));
 }
 
+//////////////////////////////////////
+// Color-Notify
+// CODE	   COLOR
```
Comment (ErikNovak) [inline src/glib/base/ut.cpp:140]: Change tabs into spaces.

File: src/glib/base/ut.cpp [hunk 2]
```diff
@@ -129,12 +129,45 @@ void TStdErrNotify::OnStatus(const TStr& MsgStr){
...
+// [01;33m	Bold Yellow
+// [0;34m	 Blue
+// [1;34m	 Bold Blue
+// [0;35m	 Magenta
+// [1;35m	 Bold Magenta
+// [0;36m	 Cyan
+// [1;36m	 Bold Cyan
+// [0m	   Reset
+
+void TColorNotify::OnStatus(const TStr& MsgStr) {
```
Comment (ErikNovak) [inline src/glib/base/ut.cpp:155]: Requires consistent number of indentation spaces. The rest of the code has 4 spaces.

File: src/glib/base/ut.cpp [hunk 3]
```diff
@@ -129,12 +129,45 @@ void TStdErrNotify::OnStatus(const TStr& MsgStr){
 // Log-Notify
 void TLogNotify::OnStatus(const TStr& MsgStr) {
 	TTm NowTm = TTm::GetCurLocTm();
-	Notify->OnStatus(TStr::Fmt("[%s %s] %s",
```
Comment (ErikNovak) [inline src/glib/base/ut.cpp:132]: Change tabs into 4 spaces.

File: src/glib/base/ut.h [hunk 4]
```diff
@@ -190,6 +190,31 @@ class TLogNotify : public TNotify {
 	void OnStatus(const TStr& MsgStr);
 };
 
+//////////////////////////////////////
+// Color-Notify
+enum class TColorNotifyType {
```
Comment (ErikNovak) [inline src/glib/base/ut.h:195]: Requires consistent number of indentation spaces. The rest of the code has 4 spaces.

**Review by ErikNovak — CHANGES_REQUESTED** [review]
Requested spacing changes:
- Tabs to spaces.
- Consistent number of spacing throughout the file (4 spaces for indentation).

**Review by ErikNovak — APPROVED** [review]

**Pull request merged**